src/api/Jalasoft.TeamUp.Resumes.API/Startup.cs

```diff
@@ -19,7 +19,11 @@ public override void Configure(IFunctionsHostBuilder builder)
             builder.Services.AddScoped<IHealthService, HealthService>();
             builder.Services.AddScoped<IHealthRepository, HealthRepository>();
             builder.Services.AddScoped<IResumesService, ResumesService>();
-            builder.Services.AddScoped<IRepository<Resume>, ResumesRepository>();
+#if DEBUG
+            builder.Services.AddScoped<IRepository<Resume>, ResumeSQLRepository>();
+#else
+            builder.Services.AddScoped<IRepository<Resume>, ResumesInMemoryRepository>();
+#endif
```

Comment:
please sync with @lina-ramirez-jalafund about this, she has a PR https://github.com/fundev3/teamup-resumes/pull/14 doing the same as yours

Reply:
Yes, we work, in the same branch, in parallel because our task is dependent, and waiting for the changes to be made in the development, will delay the delivery of this feature.